core/templates/pages/contributor-dashboard-page/translation-language-selector/review-translation-language-selector.component.ts

```diff
@@ -113,12 +113,12 @@ export class ReviewTranslationLanguageSelectorComponent implements OnInit {
   }
 
   populateLanguageSelection(languageCode: string): void {
-    this.setActiveLanguageCode.emit(languageCode);
```

Comment:
I would think we would still want to emit the active language code event change when the preferred language is set.

Reply:
Yes.